src/artemis/devices/system_tests/test_gridscan_system.py

```diff
@@ -15,8 +15,46 @@ def fast_grid_scan():
 
 
 @pytest.mark.s03
-def test_set_program_data_and_kickoff(fast_grid_scan: FastGridScan):
+def test_can_set_program_data_and_expected_images_correct(fast_grid_scan: FastGridScan):
     RE = RunEngine()
     RE(set_fast_grid_scan_params(fast_grid_scan, GridScanParams(2, 2)))
-    kickoff_status = fast_grid_scan.kickoff()
-    kickoff_status.wait()
+    assert fast_grid_scan.expected_images.get() == 2 * 2
+
+
+@pytest.mark.s03
+def test_staging_fast_grid_scan_clears_position_counter(fast_grid_scan: FastGridScan):
+    RE = RunEngine()
+    RE(set_fast_grid_scan_params(fast_grid_scan, GridScanParams(2, 2)))
+    assert fast_grid_scan.expected_images.get() == 2 * 2
```

Comment:
this seems a little superfluous on top of the previous test

Reply:
Agreed, I think I was building the tests up incrementally and so had the duplicates in there. If they were fast unit tests I might argue that it's useful to have explicit separate ones but as they'll take some time to run we should keep the number of tests low.